src/types/DateTime.ts

```diff
@@ -0,0 +1,25 @@
+import * as TypeUtils from "../TypeUtils"
+import * as _ from 'lodash'
+
+export const DateTime: SBVRType<InternalDate, NullableDate> = {
+	types: {
+		postgres: 'TIMESTAMP',
+		mysql: 'TIMESTAMP',
+		websql: 'INTEGER',
+		odata: {
+			name: 'Edm.DateTime'
+		},
+	},
+
+	fetchProcessing: (data, callback) => {
+		let processedValue: InternalDate
+		if(_.isDate(data) || data == null) {
+			processedValue = data
+		} else {
+			processedValue = new Date(data as string)
```

Comment:
This `as string` should be unnecessary

Reply:
date is still `string | number` here

Reply:
The fix isn't in 2.8.3: https://github.com/Microsoft/TypeScript/blob/v2.8.3/src/lib/es5.d.ts#L775-L776